plugins/events/init.go

```diff
@@ -53,5 +60,6 @@ func (p *Plugin) RestRoutes() route.Routes {
 // RegisterStepFuncs registers transaction step functions with
 // Glusterd Transaction framework
 func (p *Plugin) RegisterStepFuncs() {
+	registerWebhookTestStepFuncs()
```

Comment:
This confuses me. Why does webhooks support need transaction steps?

Reply:
Transaction steps is required since this needs to be run in multiple nodes